packages/verkle/test/verkle.spec.ts

```diff
@@ -211,7 +211,7 @@ describe('Verkle tree', () => {
     assert.deepEqual(val2, hexToBytes(values[2]), 'confirm values[2] can be retrieved from trie')
   })
 
-  it('should put values and find them', async () => {
+  it('should put, find, delete, and the put values (again)', async () => {
```

Comment:
I'm not sure I get the part that says `and the put values (again)`. Could you clarify?

Reply:
From looking at the test I adjusted the description of the test, feel free to adjust.

Reply:
Above is fine.  I just wanted to make sure that "deleting" and then putting in the same spot again still worked correctly.